storage/time-series/influxdb/retriever/app/influx/DataQuery.js

```diff
@@ -240,8 +320,28 @@ class DataQuery {
   static commonQueryParams(page, filters) {
     const limit = page.limit ? fluxInteger(page.limit) : 256;
     const pageNumber = page.page && page.page >= 1 ? fluxInteger(page.page) : 1;
-    const start = filters.dateFrom ? fluxDateTime(filters.dateFrom) : 0;
-    const stop = filters.dateTo ? fluxDateTime(filters.dateTo) : fluxExpression('now()');
+    /*  Start is the earliest time to include in results.
```

Comment:
Please revert changes to commonQueryParams and create a separate method for your use case. To avoid side effects of problems with what is already working.